website/docs/cloud-docs/integrations/kubernetes/api-reference.mdx

```diff
@@ -584,6 +584,7 @@ _Appears in:_
 | `oAuthTokenID` _string_ | The VCS Connection (OAuth Connection + Token) to use. Must match pattern: ^ot-[a-zA-Z0-9]+$ |
 | `repository` _string_ | A reference to your VCS repository in the format `<organization>/<repository>` where `<organization>` and `<repository>` refer to the organization and repository in your VCS provider. |
 | `branch` _string_ | The repository branch that Run will execute from. This defaults to the repository's default branch (e.g. main). |
+| `speculativePlans` _boolean_ | Whether this workspace allows automatic speculative plans on PR. Default: `true`. [More information](https://developer.hashicorp.com/terraform/cloud-docs/run/ui#speculative-plans-on-pull-requests) |
```

Comment:
```suggestion
| `speculativePlans` _boolean_ | Allow automatic speculative plans on pull requests. Default: `true`. [More information](https://developer.hashicorp.com/terraform/cloud-docs/run/ui#speculative-plans-on-pull-requests) |
```

Reply:
I know this page is automatically generated, so if it's easier, you can decline this suggestion and we can leave it as-is for the release, and then I can take a swing at opening a PR on the codebase to update some of the docs. I think we discussed doing this in the past, so I'm certainly open to it!

Reply:
If that is fine for you, let's leave it as it is, and in a few weeks, I plan to bump up the version of the tool that generates the API reference. The have made some significant changes there and now "Default" and "Validation" are dedicated columns that make it more user-friendly. So it will be a significant change and I will include your suggestion there too.